pallets/gear/src/benchmarking/mod.rs

```diff
@@ -278,6 +284,10 @@ benchmarks! {
         T::AccountId: Origin,
     }
 
+    check_syscalls_integrity {
```

Comment:
I guess we should put it under feature or it will fall in WeightInfo other way

Reply:
What's the case with 'WeightInfo'? For now it works ok with both executors and all the features.

Reply:
discussed, resolved